Bottom: b1b3d52468db9813f5033165f6b2bb6db8ab202d
Top:    35a15924968b5943184f1504b81e79e8d23d3a55
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-15 12:31:37 +0000

Refresh of play-with-perspective

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 74499a1..9635e46 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -432,7 +432,6 @@ ghid_draw_acy_resistor (ElementType *element)
     float z;
     float r;
 
-
     z = -resistor_width / 2.;
     r = resistor_pin_radius;
     /* repeat first vertex */
